src/adapters/chimp-exchange/products/pool/chimpExchangePoolAdapter.ts

```diff
@@ -175,9 +176,7 @@ export class ChimpExchangePoolAdapter
         const tokensData = await filterMapAsync(
           event.args.tokens,
           async (token, index) => {
-            if (
-              token.toLowerCase() === protocolTokenAddress.toLocaleLowerCase()
-            ) {
+            if (getAddress(token) === getAddress(protocolTokenAddress)) {
```

Comment:
I think `protocolTokenAddress` here should already be checksumed, as it's input from defiProvider.